### ISSUE TITLE
Mob ran over trap without activating it

### ISSUE THREAD

**Issue by bcp19** [echoes-emu/echoes-bt#684]
I think this may be location specific. Set trap, attacked wolf, wolf ran over trap without it activating.  You are at bx66 by119 c0 ox149.043 oy102.532 oz429.877 r126 (Instance 0)
![ScreenShot00054](https://user-images.githubusercontent.com/122328856/211647223-f23460f5-7931-44c8-8397-295ee8a6b46f.jpg)

**Comment by woodx319** [comment]
Yeah I wonder if terrain makes the whole circle not be able to deploy causes this 

- [ ] Test terrain hills+traps

**Comment by woodx319** [comment]
I did this a few times and most times it got stuck about a third of the time it didn't trap. other spots around the area seem fine. Never mained hunter so idk how important the entire red circle being down is for good trap placement
